api/src/main/java/com/gentics/mesh/util/PasswordUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.gentics.mesh.util;
+
+import java.util.Random;
+
+/**
+ * Utility class which provides methods to handle passwords.
+ */
+public final class PasswordUtil {
+
+	private static final String ALPHA_NUMERIC_READABLE_CHAR = "ABCDEFGHJKLMNPQRTUVWXYZ"
+		+ "2346789"
+		+ "abcdefghjkmnpqrtuvxyz";
```

Comment:
`pwgen` uses these characters:
```
const char *pw_digits = "0123456789";
const char *pw_uppers = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
const char *pw_lowers = "abcdefghijklmnopqrstuvwxyz";
const char *pw_symbols = "!\"#$%&'()*+,-./:;<=>?@@[\\]^_`{|}~";
const char *pw_ambiguous = "B8G6I1l0OQDS5Z2";
```
Remove the ambiguous characters from the set.

Or you can rely on the fact that the password will be copy & pasted anyways.